modules/claims_api/spec/fixtures/v2/veterans/disability_compensation/form_526_json_api.json

```diff
@@ -5,24 +5,24 @@
             "veteranIdentification": {
                 "currentlyVaEmployee": false,
                 "mailingAddress": {
-                    "numberAndStreet": ",a#'",
-                    "apartmentOrUnitNumber": "ABCD",
-                    "city": "#j2'w",
-                    "country": "ABCDEFGHIJKLMNOPQRSTUVW",
+                    "numberAndStreet": "123 Easy Street",
```

Comment:
Just noting I populated most of these values during implementation of the section 1 validation merged yesterday. Feel free to overwrite my values with yours when you merge.